specification.markdown

```diff
@@ -97,242 +170,298 @@ followed by the Data Size, and then the non-interpreted Binary itself:
-    damaged data. The content is discarded. Also used to reserve space
-    in a sub-element for later use.</td>
-  </tr>
-  <tr class="subclass">
-   <td colspan="9">signature</td>
-  </tr>
-  <tr class="version2">
-   <td>SignatureSlot<a name="SignatureSlot"></a></td>
-   <td>1+</td>
-   <td>[1B][53][86][67]</td>
-   <td>-</td>
-   <td>*</td>
-   <td>-</td>
-   <td>-</td>
-   <td>sub-elements</td>
-   <td>Contain signature of some (coming) elements in the stream.</td>
-  </tr>
-  <tr class="version2">
-   <td>SignatureAlgo<a name="SignatureAlgo"></a></td>
-   <td>2+</td>
-   <td>[7E][8A]</td>
-   <td>-</td>
-   <td>-</td>
-   <td>-</td>
-   <td>-</td>
-   <td>u-integer</td>
-   <td>Signature algorithm used (1=RSA, 2=elliptic).</td>
-  </tr>
-  <tr class="version2">
-   <td>SignatureHash<a name="SignatureHash"></a></td>
-   <td>2+</td>
-   <td>[7E][9A]</td>
-   <td>-</td>
-   <td>-</td>
-   <td>-</td>
-   <td>-</td>
-   <td>u-integer</td>
-   <td>Hash algorithm used (1=SHA1-160, 2=MD5).</td>
-  </tr>
-  <tr class="version2">
-   <td>SignaturePublicKey<a name="SignaturePublicKey"></a></td>
-   <td>2+</td>
-   <td>[7E][A5]</td>
-   <td>-</td>
-   <td>-</td>
-   <td>-</td>
-   <td>-</td>
-   <td>binary</td>
-   <td>The public key to use with the algorithm (in the case of a PKI-based
-    signature).</td>
-  </tr>
-  <tr class="version2">
-   <td>Signature<a name="Signature"></a></td>
-   <td>2+</td>
-   <td>[7E][B5]</td>
-   <td>-</td>
-   <td>-</td>
-   <td>-</td>
-   <td>-</td>
-   <td>binary</td>
-   <td>The signature of the data (until a new.</td>
-  </tr>
-  <tr class="version2">
-   <td>SignatureElements<a name="SignatureElements"></a></td>
-   <td>2+</td>
-   <td>[7E][5B]</td>
-   <td>-</td>
-   <td>-</td>
-   <td>-</td>
-   <td>-</td>
-   <td>sub-elements</td>
-   <td>Contains elements that will be used to compute the signature.</td>
-  </tr>
-  <tr class="version2">
-   <td>SignatureElementList<a name="SignatureElementList"></a></td>
-   <td>3+</td>
-   <td>[7E][7B]</td>
-   <td>-</td>
-   <td>*</td>
-   <td>-</td>
-   <td>-</td>
-   <td>sub-elements</td>
-   <td>A list consists of a number of consecutive elements that represent one
-    case where data is used in signature. Ex: <i>Cluster|Block|BlockAdditional</i>
-    means that the BlockAdditional of all Blocks in all Clusters is used for
-    encryption.</td>
-  </tr>
-  <tr class="version2">
-   <td>SignedElement<a name="SignedElement"></a></td>
-   <td>4+</td>
-   <td>[65][32]</td>
-   <td>-</td>
-   <td>*</td>
-   <td>-</td>
-   <td>-</td>
-   <td>binary</td>
-   <td>An element ID whose data will be used to compute the signature.</td>
-  </tr>
- </tbody>
-</table>
+## Structure
+
+Syntactically an EBML document is a list of EBML elements. Each element
+has three parts; an Element ID, an Element Data Size and the element
+data payload. The element data payload is then either data, implicitly
+typed by the Element ID, or a list of EBML elements.
+
+EBML       = *ELEMENT
```

Comment:
fixed in https://github.com/MediaArea/ebml-specification/commit/a3ae17f4d8e77295ff899a2b7a4a0923c6105e12